js-exercises/remove-falsy-values/removeFalsyValues.js

```diff
@@ -0,0 +1,8 @@
+
+function removeFalsyValues(array) {
+  return array.filter(value => Boolean(value));
```

Comment:
`filter(value => Boolean(value))` is just the same as `filter(Boolean)`

Reply:
Fixed this comment and committed code.